packages/client/wallets/smart-wallet/src/blockchain/rpc.ts

```diff
@@ -0,0 +1,18 @@
+import { SmartWalletChain } from "./chains";
+
+const ALCHEMY_API_KEY = "-7M6vRDBDknwvMxnqah_jbcieWg0qad9";
```

Comment:
I hate this, but I don't think we can get around it trivially

Reply:
Wonder if we can do project and ip specific rate limits here